test/specs/common/tests/02.column.js

```diff
@@ -0,0 +1,194 @@
+
+            describe('the .formatvalue property, ', function() {
+                var formatUtils = options.includes.ermRest._formatUtils;
+
+                it('should be empty string if column value is null.', function() {
+                    var column = table1_schema2.columns.getByPosition(2);
+                    expect(column.formatvalue(null)).toEqual(jasmine.any(String));
+                    expect(column.formatvalue(null)).toBe('');
+                });
+
+                describe('should call printText() to format,', function() {
+                    var formattedValue = undefined;
+
+                    it('text columns correctly.', function() {
+                        var spy = spyOn(formatUtils, 'printText').and.callThrough();
+                        var textCol = table1_schema2.columns.getByPosition(2);
+                        formattedValue = textCol.formatvalue('text value');
+                        expect(spy).toHaveBeenCalledWith('text value', undefined);
+                        expect(formattedValue).toBe('text value');
+                    });
+
+                    it('longtext columns correctly.', function() {
+                        var spy = spyOn(formatUtils, 'printText').and.callThrough();
+                        var longtextCol = table1_schema2.columns.getByPosition(3);
+                        var longtext = 'asjdf;laksjdf;laj ;lkajsd;f lkajsdf;lakjs f;lakjs df;lasjd f;ladsjf;alskdjfa ;lskdjf a;lsdkjf a;lskdfjal;sdkfj as;ldfkj as;dlf kjasl;fkaj;lfkjasl;fjas;ldfkjals;dfkjas;dlkfja;sldkfjasl;dkfjas;dlfkjasl;dfkja; lsdjfk a;lskdjf a;lsdfj as;ldfja;sldkfja;lskjdfa;lskdjfa;lsdkfja;sldkfjas;ldfkjas;dlfkjas;lfkja;sldkjf a;lsjf ;laskj fa;slk jfa;sld fjas;l js;lfkajs;lfkasjf;alsja;lk ;l kja';
+                        formattedValue = longtextCol.formatvalue(longtext);
+                        expect(spy).toHaveBeenCalledWith(longtext, undefined);
+                        expect(formattedValue).toBe(longtext);
+                    });
+
+                    afterEach(function() {
+                        expect(formattedValue).toEqual(jasmine.any(String));
+                        formattedValue = undefined;
+                    });
+                });
+
+                describe('should call printDate() to format,', function() {
+                    it('date columns correctly.', function() {
+                        var spy = spyOn(formatUtils, 'printDate').and.callThrough();
+                        var testVal = '2016-05-02 13:00:00.00 PST';
+                        var col = table1_schema2.columns.getByPosition(4);
+                        var options = {separator: '.', leadingZero: false};
+                        var formattedValue = col.formatvalue(testVal, options);
+                        expect(spy).toHaveBeenCalledWith(testVal, options);
+                        expect(formattedValue).toEqual(jasmine.any(String));
+                        expect(formattedValue).toBe('2016.5.2');
+                    });
```

Comment:
The default separator is a slash (`/`), but I created an option to specify your own separator if desired.